theano/compile/ops.py

```diff
@@ -623,6 +603,7 @@ class Rebroadcast(gof.Op):
     c_code_and_version = {}
 
     check_input = False
+    __props__ = ("axis")
 
     def __init__(self, *axis):
         self.axis = dict(axis)
```

Comment:
In \__hash__ axis was sorted. I think we should use an OrderedDict, that we create empty and add the axis to it in the sorted order. That would make this Op better and allow to use \__props__ without regression.